README.markdown

```diff
@@ -117,9 +119,8 @@ Results in:
 
 You might want to create child providers that inherit the `ini_setting` provider, for one or both of these purposes:
```

Comment:
I think we could change "for one or both..." to "if"

Reply:
Oh, "if" won't really go with the way you've revised the bullet points, but maybe just make this bit more direct. "for these purposes" feels indirect and wordy.